packages/rs-drive-abci/src/execution/platform_events/withdrawals/update_broadcasted_withdrawal_statuses/mod.rs

```diff
@@ -35,25 +35,25 @@ where
     /// * There is an issue interacting with the underlying storage.
     /// * There is an error fetching the withdrawal data contract.
     /// * There is an error getting the transactionId or transactionSignHeight from the withdrawal document.
-    pub fn update_broadcasted_withdrawal_transaction_statuses(
+    pub fn update_broadcasted_withdrawal_statuses(
         &self,
-        block_execution_context: &BlockExecutionContext,
-        transaction: &Transaction,
+        block_info: &BlockInfo,
+        transaction: TransactionArg,
```

Comment:
This can only be on a transaction, don't replace it with an optional transaction.